diff --git a/libs/starlight.ts b/libs/starlight.ts
index 8980040c4adebaca54dadecb513686b9f3409fa9..44bea5b46ca876b36b5225bc10c24422eab43bde 100644
--- a/libs/starlight.ts
+++ b/libs/starlight.ts
@@ -165,11 +165,14 @@ function addPrefixToSlug(prefix: string, slug: string) {
 
 type StarlightFrontmatter = z.input<ReturnType<ReturnType<typeof docsSchema>>> & {
   slug?: string
+  prev?: boolean | string | { link?: string; label?: string }
+  next?: boolean | string | { link?: string; label?: string }
   hero?: StarlightFrontmatteHero
 }
 
 // Just to simplify type checking when updating the frontmatter,
 interface StarlightFrontmatteHero {
+  actions?: { link: string; text: string }[]
   image?: {
     file?: string
     dark?: string
